src/Components/App/App.js

```diff
@@ -1,19 +1,19 @@
+import React from 'react';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom'; 
 import './App.css';
+import HomePage from '../HomePage/HomePage'; 
+import ErrorPage from '../ErrorPage/ErrorPage'; 
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Router>
+      <div className="App">
+        <Routes>
+          <Route path="/" element={<HomePage />} />
+          <Route path="/error" element={<ErrorPage />} />
```

Comment:
We are planning to conditionally render this error page, correct? At the top of the App return?

Reply:
@andreasorensen  yes we are good catch!